fixup-exports.py

```diff
@@ -1,160 +0,0 @@
-import argparse
-import configparser
-import os
-import pathlib
-import shutil
-import sys
-import xml.etree.cElementTree as xml_etree
-
-
-__dir__ = pathlib.Path(__file__).parent
-__version__ = "1"
-
-
-def rename_app_icon(prefix, from_name, to_name):
-	xdg_icons_dir = prefix / "share" / "icons"
-	
-	kill_list = set()
-	
-	for xdg_icons_app_path in xdg_icons_dir.glob("*/*/apps/"):
-		for icon_path in map(lambda v: xdg_icons_app_path / v.name, os.scandir(xdg_icons_app_path)):
-			if icon_path.with_suffix("").name == from_name:
-				shutil.copy2(icon_path, icon_path.with_name(to_name + icon_path.suffix))
-				kill_list.add(icon_path)
-	
-	return kill_list
-
-
-def rename_desktop_with_icon(prefix, from_name, to_name):
-	xdg_applications_dir = prefix / "share" / "applications"
-	
-	kill_list = set()
-	
-	# Read original desktop file
-	desktop_file = configparser.ConfigParser(
-		allow_no_value          = False,
-		delimiters              = ("=",),
-		comment_prefixes        = ("#",),
-		inline_comment_prefixes = None,
-		interpolation           = None,
-		strict                  = False,
-		empty_lines_in_values   = False,
-	)
-	desktop_file.optionxform = lambda option: option  # Prevent lower-casing of option names
-	assert desktop_file.read(xdg_applications_dir / f"{from_name}.desktop", encoding="utf-8")
-	
-	# Copy referenced icon to acceptable file name
-	icon_name = desktop_file["Desktop Entry"].get("Icon")
-	if icon_name is not None and icon_name != to_name:
-		kill_list |= rename_app_icon(prefix, icon_name, to_name)
-		desktop_file["Desktop Entry"]["Icon"] = icon_name = to_name
-	
-	# Write back new (possibly modified) .desktop file
-	with open(xdg_applications_dir / f"{to_name}.desktop", "w", encoding="utf-8") as f:
-		desktop_file.write(f, space_around_delimiters=False)
-	
-	# Queue previous desktop file for removal
-	if from_name != to_name:
-		kill_list.add(xdg_applications_dir / f"{from_name}.desktop")
-	
-	return kill_list
-
-
-def rename_appdata_with_desktop(prefix, from_name, to_name):
-	xdg_metainfo_dir = prefix / "share" / "metainfo"
-	
-	kill_list = set()
-	
-	appdata = xml_etree.parse(xdg_metainfo_dir / f"{from_name}.metainfo.xml")
-	
-	extra_provides = set()
-	
-	# Record change of appid encoded in file name
-	if from_name != to_name and '.' in from_name:
-		extra_provides.add(from_name if '.' in from_name else f"{from_name}.desktop")
-	
-	# Update AppID encoded into the XML ID element (must match the filename)
-	appid_elem = appdata.find("./id")
-	if appid_elem is not None:
-		app_id = appid_elem.text
-		if app_id != to_name:
-			extra_provides.add(app_id)
-			appid_elem.text = app_id = to_name
-	
-	# Update desktop file location to match new appid/`to_name`
-	for launchable_elem in appdata.findall("./launchable[@type='desktop-id']"):
-		desktop_id = launchable_elem.text
-		if desktop_id.endswith(".desktop"):
-			desktop_id = desktop_id[:-len(".desktop")]
```

Comment:
This needs to be incorporated or the app will not be launchable from software managers after installation. (See other comment.)

Reply:
Done with `appstream-util modify launchable`